### ISSUE TITLE
Create Cloud Service window is getting opened when the user presses the down key after selecting the drop down button of cloud service

### ISSUE THREAD

**Issue by kasturiswain** [MSOpenTech/msopentech-tools-for-intellij#184]
<b>Environment: </b>Windows/IntelliJ14

Steps:
While creating the new VM In the Cloud Service settings window, click on the drop down below the cloud service.
Press the down key from the key board.
Is should not open the create cloud service window unless the user presses the ENTER key after the down key from the key board.

**Comment by avranju** [comment]
@leotilli Can we add a comment to indicate why this is classified as *wontfix*?
